src/cli/actions/hub/login.js

```diff
@@ -63,10 +66,21 @@ async function login () {
   const tokenUrl = `${hostname}/oauth/token`
 
   try {
-    const response = await axios.post(deviceCodeUrl, {
-      client_id: OAUTH_CLIENT_ID
+    const response = await request(deviceCodeUrl, {
+      method: 'POST',
+      body: JSON.stringify({ client_id: OAUTH_CLIENT_ID })
```

Comment:
it's failing here.

can you please create an account on hub and then run locally to test out.

Reply:
returning

```
{"error":"invalid_grant","error_description":"that device_code is invalid, expired, revoked, not found, or was issued to a different client"}
```

from hub.dotenvx.com's api.

Reply:
hmm weird will check

Reply:
works for me now, had to include the content type header